src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Logging/LoggerMessageState.ClassifiedProperty.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Compliance.Classification;
+
+namespace Microsoft.Extensions.Telemetry.Logging;
+
+public partial class LoggerMessageState
```

Comment:
Even though loggers are recommended to use the formatter delegate, I believe some of them just call ToString() instead. All of the built-in TStates that .NET generates implement ToString() so that produces the same string you would have gotten if you called formatter(state, null);

Reply:
Added ToString implementation to the LoggerMessageState.